flake.nix

```diff
@@ -0,0 +1,44 @@
+{
+  description = "Dev environment";
+
+  outputs = { self, nixpkgs }:
+    let pkgs = nixpkgs.legacyPackages.x86_64-linux;
+    in {
+      defaultPackage.x86_64-linux =
+        pkgs.mkShell {
+          buildInputs =  with pkgs; [
+            (php82.withExtensions
+              ({ all, ... }: with all; [
+                bz2
+                curl
+                dom
+                filter
+                fileinfo
+                gd
+                iconv
+                imagick
+                intl
+                mbstring
+                openssl
+                pdo
+                pdo_mysql
+                pdo_sqlite
+                session
+                sodium
+                sqlite3
+                tidy
+                tokenizer
+                xdebug
+                xmlwriter
+#                yaml
+                zip
+                zlib
+              ])
+            )
+            php82Packages.composer
+            libjpeg
+            libpng
+            libyaml
```

Comment:
What are these used for?

Reply:
I've copied the flake from my own project where I need those - here they're not needed. I removed them.